src/sections/Blog/Blog-grid/index.js

```diff
@@ -12,7 +12,7 @@ import { BlogPageWrapper } from "./blogGrid.style";
 const BlogGrid = ({data, isListView, setListView, setGridView, pageContext}) => {
   return (
     <BlogPageWrapper>
-      <PageHeader title="Blog" path="Blog"/>
+      <PageHeader title="Resources" path="Blog"/>
```

Comment:
@ShivamTyagi12345 You are changing the heading on the blog  page, i think you have to change the navbar.

Reply:
@ShivamTyagi12345 I tried when we change the link in the dropdown , it doesn't drop any error but the error shows when we change from the main navbar, from what i found was the `active.name`  contain the "blog" link somewhere because of which it shows null , i think you can solve the error by looking into `/src/sections/General/Navigation/index.js` or `/src/sections/General/Navigation/utility/ScrollspyMenu.js` (tried for hours but couldn't find the issue)